packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -409,13 +409,27 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                     let cachedSnapshot: ISnapshotContents | undefined;
                     const cachedSnapshotP: Promise<ISnapshotContents | undefined> =
                         this.epochTracker.get(createCacheSnapshotKey(this.odspResolvedUrl))
-                            .then((snapshotCachedEntry: ISnapshotCachedEntry) => {
+                            .then(async (snapshotCachedEntry: ISnapshotCachedEntry) => {
                                 if (snapshotCachedEntry !== undefined) {
-                                    // If the cached entry does not contain the entry time, then assign it a default of 30 days old.
-                                    // eslint-disable-next-line @typescript-eslint/dot-notation
-                                    props["cacheEntryAge"] = Date.now() - (snapshotCachedEntry.cacheEntryTime ??
+                                    // If the cached entry does not contain the entry time, then assign it a default of 2 days old.
+                                    const age = Date.now() - (snapshotCachedEntry.cacheEntryTime ??
                                         (Date.now() - 30 * 24 * 60 * 60 * 1000));
+
+                                    // eslint-disable-next-line @typescript-eslint/dot-notation
+                                    props["cacheEntryAge"] = age;
+
+                                    // TODO: Permanently enable this with a default of 2 days.
+                                    // TODO: Refactor this (maybe move to a separate function)
+                                    if(hostSnapshotOptions?.maxSessionActiveTime) {
```

Comment:
Please do not convert number | undefined to falsy. 0 is falsy, so someone putting zero (I'd guess assuming that would disable cache) would do nothing. Maybe zero is not great number to use (as we waste resources by caching and never using it), but I think you should be explicit here and compare against undefined